src/ConnectionLocator.php

```diff
@@ -77,18 +146,27 @@ public function setWriteFactory(
         $this->writeFactories[$name] = $factory;
     }
 
+    /**
+     * @return Connection
+     */
     public function getDefault() : Connection
     {
-        if ($this->instances[static::DEFAULT] === null) {
-            $this->instances[static::DEFAULT] = $this->newConnection(
+        /** @var 'DEFAULT' $type */
+        $type = static::DEFAULT;
+
+        if ($this->instances[$type] === null) {
+            $this->instances[$type] = $this->newConnection(
                 $this->defaultFactory,
-                static::DEFAULT
+                $type
             );
```

Comment:
This was the only way I could find to soothe stan.

Reply:
I'm good with that!